tests/Unit/Service/Security/SensioLabsPackageScannerTest.php

```diff
@@ -128,10 +131,16 @@ private function prepareScanner(string $fixtureType = 'repman'): SensioLabsPacka
 
         $packageManager->method('distFilename')->willReturn(Option::some($distFile));
 
+        $messageBusMock = $this->createMock(MessageBus::class);
+        $messageBusMock
+            ->method('dispatch')
+            ->willReturn(new Envelope(new SendScanResult('buddy', 'test/test', 'test', [])));
```

Comment:
`willReturn` is unused in this case, better you some kind of assertion: `expects($this->once())`

Reply:
this is actually done to supress warnings (Envelope cannot be mocked) since dispatch() method must return Envelope